src/components/BackButton.js

```diff
@@ -0,0 +1,15 @@
+import Button from '../components/Button'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+BackButton.propTypes = {
+  onClick: PropTypes.func.isRequired,
+}
+
+export default function BackButton(props) {
+  return <BackButtonStyle {...props}>&lt; Back</BackButtonStyle>
+}
+const BackButtonStyle = styled(Button)`
+  margin: 10px;
+  width: 30%;
+`
```

Comment:
Do you really need an extra component for BackButton? If you make your Button component more flexible it could improve code

Reply:
Good point!